src/main/resources/swagger/api-docs.yaml

```diff
@@ -864,6 +864,103 @@ paths:
             - email
             - profile
 
+  /api/workflows/{version}/{id}/releaseHold:
+    post:
+      x-passthrough: true
+      x-passthrough-target: cromiam
+      summary: Request Cromwell to release the hold on a workflow. It will switch the status of a workflow
+        from ‘On Hold’ to ‘Submitted’ so it can be picked for running. For instance this might be necessary
+        in cases where you have submitted a workflow with workflowOnHold = true.
+      parameters:
+        - name: version
+          description: API Version
+          required: true
+          type: string
+          in: path
+          default: v1
+        - name: id
+          description: Workflow ID
+          required: true
+          type: string
+          in: path
+      tags:
+        - CromIAM Workflows (for Admin)
+      responses:
+        '200':
+          description: Successful Request
+          schema:
+            $ref: '#/definitions/WorkflowStatusResponse'
+        '400':
+          description: Malformed Workflow ID
+        '403':
+          description: Malformed Request
+        '404':
+          description: Workflow ID Not Found
+        '500':
+          description: Internal Error
+      security:
+        - googleoauth:
+            - openid
+            - email
+            - profile
+
+  '/api/workflows/{version}/callcaching/diff':
+    get:
+      summary: Return the hash differential between two calls
+      parameters:
+        - name: version
+          description: API Version
+          required: true
+          type: string
+          in: path
+          default: v1
```

Comment:
not a blocker for this PR: the `version` param is used for a lot of APIs and could/should be made into a reusable param in the `parameters` block  around line 5162.